### DISCUSSION TITLE
Add Client with same name but different ip address

### DISCUSSION THREAD

**Discussion by chatbuddy21** [AdguardTeam/AdGuardHome#4020] [Q&A] [answered]
I'm trying to add Client with the same name but different ip address, I'm getting an error that it already exists. I wonder why this is not supported?

Error: control/clients/add | Client already exists | 400

Google-Home-Mini - 192.168.1.2
Google-Home-Mini - 192.168.1.3

**Comment by ppfeufer** [comment, marked as answer]
Click on the `+` to add another IP
![image](https://user-images.githubusercontent.com/2989985/147410153-e045c608-3d90-4996-97d1-c17465394ec6.png)

**Reply by chatbuddy21** [reply]
Thanks!
